cmd/zed/import.go

```diff
@@ -140,13 +142,34 @@ func importRelationships(client *authzed.Client, relationships string) error {
 	if err := scanner.Err(); err != nil {
 		return err
 	}
+	log.Info().Int("count", len(relationshipUpdates)).Msg("parsed relationships")
 
-	request := &v1.WriteRelationshipsRequest{Updates: relationshipUpdates}
-	log.Trace().Interface("request", request).Msg("writing relationships")
-	log.Info().Int("count", len(relationshipUpdates)).Msg("importing relationships")
+	return batchRequests(client, relationshipUpdates, importBatchSize)
+}
 
-	if _, err := client.WriteRelationships(context.Background(), request); err != nil {
-		return err
+func batchRequests(client *authzed.Client, updates []*v1.RelationshipUpdate, batchSize int) error {
+	totalBatches := (len(updates) + batchSize - 1) / batchSize
```

Comment:
I changed the calculation for getting the total number of batches required. The last one was unintuitive.